editor/plugins/sprite_frames_editor_plugin.cpp

```diff
@@ -506,25 +531,27 @@ void SpriteFramesEditor::_prepare_sprite_sheet(const String &p_file) {
 		EditorNode::get_singleton()->show_warning(TTR("Unable to load images"));
 		ERR_FAIL_COND(texture.is_null());
 	}
+
 	frames_selected.clear();
 	selected_count = 0;
 	last_frame_selected = -1;
+	split_data = EditorSettings::get_singleton()->get_project_metadata("sprite_frames", "split_data", Dictionary());
 
-	bool new_texture = texture != split_sheet_preview->get_texture();
+	// bool new_texture = texture != split_sheet_preview->get_texture();
```

Comment:
```suggestion
	//bool new_texture = texture != split_sheet_preview->get_texture();
```
Commented out lines are not leading spaces, though this line should really be removed